installing/installing_vsphere/upi/upi-vsphere-installation-reqs.adoc

```diff
@@ -16,6 +16,7 @@ include::modules/vmware-csi-driver-reqs.adoc[leveloffset=+1]
 
 * To remove a third-party vSphere CSI driver, see xref:../../../storage/container_storage_interface/persistent-storage-csi-vsphere.adoc#persistent-storage-csi-vsphere-install-issues_persistent-storage-csi-vsphere[Removing a third-party vSphere CSI Driver].
 * To update the hardware version for your vSphere nodes, see xref:../../../updating/updating_a_cluster/updating-hardware-on-nodes-running-on-vsphere.adoc#updating-hardware-on-nodes-running-on-vsphere[Updating hardware on nodes running in vSphere].
+* To list of minimum permissions for CSI and its components, see the "Minimum permissions for the storage components" table in the xref:../../installing_vsphere/upi/upi-vsphere-installation-reqs#installation-vsphere-installer-infra-requirements_upi-vsphere-installation-reqs[Required vCenter account privileges] section.
```

Comment:
Please refer to my comment in `installing/installing_vsphere/ipi/ipi-vsphere-installation-reqs.adoc`, which also applies here :)

Reply:
OK. I added IDs and now the xref points directly to the table. All works well in the preview.